### DISCUSSION TITLE
Enable Logging

### DISCUSSION THREAD

**Discussion by boukmi** [tada5hi/typeorm-extension#261] [General]
Hi,

Is there a way to enable logging? Is there any logs that are produced by this package?
It would be very helpful if we could see the SQL queries that are being executed against the database.

Thank you

**Comment by simon-schuster** [comment]
Hi,

You probably need to enable logging via the ormconfig. If you don't know how this is done, here is the official TypeORM doc for setting up the logging. https://typeorm.io/logging

Kind regards,
Simon

**Reply by boukmi** [reply]
Thank you @simon-schuster, I'll take a look at this.

Bests,
